DQM/DTMonitorModule/python/dtDQMOfflineSources_Cosmics_cff.py

```diff
@@ -42,11 +46,12 @@
 from DQM.DTMonitorModule.dtTriggerEfficiencyTask_cfi import *
 
 dtSourcesCosmics = cms.Sequence(dtDataIntegrityUnpacker  +
-                                DTDataIntegrityTask +
-                                dtDCSByLumiMonitor + 
-                                dtRunConditionVar + 
-                                dtSegmentAnalysisMonitor +
-                                dtResolutionAnalysisMonitor +
-                                dtEfficiencyMonitor +
-                                dtTriggerEfficiencyMonitor +
-                                dqmInfoDT)
+                         twinMuxStage2Digis +
```

Comment:
@dmitrijus Could you anyway trigger the Jenkins tests with the last commit which may fix the situation? I'm not able to run them in my local release in due a strange error not related to this PR: 
"plugin-poisoned-CondFormatsCTPPSReadoutObjectsPlugins.so cannot open shared object file"
Thanks